### ISSUE TITLE
Email notification to admin while check in / check out 

### ISSUE THREAD

**Issue by amitforucool** [grokability/snipe-it#9309]
#### Please confirm you have done the following before posting your bug report:

- [ ] I have enabled debug mode - Yes
- [ ] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues) - Yes

**Describe the bug**

We have set up the whole new asset inventory on /snipe-It. Everything works great. The only issue which I am facing is that we are not receiving any email notification of check-in/checkout to ADMINISTRATOR however receiving test email successfully. Also, the user is getting the email notification every time but not Admins.

**To Reproduce**
1. Go to 'Asset list'
2. Click on 'checkout'
3. Email notification sent to user but not Admin
4. No error anywhere

**Expected behavior**
As per the setting/configuration set, Email notification must be triggered to user and Admin

**Screenshots**
No such Screenshot necessary

**Server (please complete the following information):**
 - Snipe-IT Version - v4.6.15 build 4011
 - OS: Windows
 - Web Server:  IIS
 - PHP Version - 7.0.31

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser  chrome 
 - Version  89.0.4389.82

![email issue snipe it](https://user-images.githubusercontent.com/80809486/111426214-54690400-871a-11eb-845c-291e16e9ac4b.jpg)

**Error Messages**
- error screen shot attached.

**Additional context**
- Is this a fresh install - fresh install 
- What OS and web server you're running Snipe-IT on - Windows server 2012 r2
- Manual installation

**Comment by haupt-pascal** [comment]
Hey,

did you clicked at the category the "send mail at checkin/checkout"? (something like this - I'm using snipeit in german :)) 
![image](https://user-images.githubusercontent.com/75530910/111440226-6f317d80-8706-11eb-9c32-754a09d34fd8.png)

There is also in settings the part "alerts" -> did you activate 'em? 
![help](https://user-images.githubusercontent.com/75530910/111440473-ad2ea180-8706-11eb-8721-3b578fc84bff.png)

**Comment by snipe** [comment]
Yeah, I think @haupt-pascal is on the right track here. If you have the Admin Settings configured correctly, it should CC that admin address you specify.

**Comment by RiderWJ** [comment]
I have same issue as yours @amitforucool Did you fixed? Almost all the settings I have are the same as @haupt-pascal .

**Comment by Manchester10196** [comment]
I have the same issue. Although I have enabled the email notification, specify CC that is the admin address and all  Admin Settings configured correctly.

**Comment by marcusmoore** [comment]
@Manchester10196 which version are you using? Is there anything appearing in your `.log` file?

**Comment by Manchester10196** [comment]
@marcusmoore **Version v6.0.14**, but what should I check in the log file?

**Comment by marcusmoore** [comment]
@Manchester10196 updating to the latest version might help since we fixed some bugs around notifications recently.

In the logs you would be looking for any exceptions or errors being recorded.

**Comment by Manchester10196** [comment]
production.ERROR: Swift_TransportException: Failed to authenticate on SMTP server with username "YOURUSERNAME" using 2 possible authenticators. Authenticator LOGIN returned Expected response code 334 but got code "530", with message "530 Must issue a STARTTLS command first
". Authenticator PLAIN returned Expected response code 235 but got code "530", with message "530 Must issue a STARTTLS command first
". in C:\xampp\htdocs\snipe-it\vendor\swiftmailer\swiftmailer\lib\classes\Swift\Transport\Esmtp\AuthHandler.php:191



@marcusmoore  **I think that is the error, any idea how to solve it?**

**Comment by marcusmoore** [comment]
@Manchester10196 `Failed to authenticate on SMTP server with username "YOURUSERNAME"` it looks like you need to set your [mail settings](https://snipe-it.readme.io/docs/configuration#required-outgoing-mail-settings).

**Comment by Manchester10196** [comment]
@marcusmoore I have already set them 
MAIL_DRIVER=smtp
MAIL_HOST=smtp.gmail.com
MAIL_PORT=587
MAIL_USERNAME=$$$$@gmail.com
MAIL_PASSWORD=$$$$
MAIL_ENCRYPTION=tls
MAIL_FROM_ADDR=$$$$@gmail.com
MAIL_FROM_NAME=' $$$$'
MAIL_REPLYTO_ADDR=$$$$@gmail.com
MAIL_REPLYTO_NAME='$$$$'
MAIL_AUTO_EMBED_METHOD='attachment'

NOTE: $$$$ HAS A VALUE

**Comment by marcusmoore** [comment]
@Manchester10196 do you get any other types of email notifications from the app? (ie: check out/ check in emails being sent to users when interacting with items whose category is set to send those types of emails)

Is that error showing up recently or is it an older message from before the email settings were set in the `.env` file? If those credentials were entered later and you are still not receiving email you might need to go through the [extra steps for sending email via Gmail](https://snipe-it.readme.io/docs/configuration#using-gmail).

**Comment by snipe** [comment]
Hi there - We haven't heard back in a bit, so I'm going to close this ticket for now, but will re-open it if you're still having issues.

![hackerman-hacking](https://user-images.githubusercontent.com/197404/31936003-27a7f6d8-b865-11e7-8f5e-eb03062925f4.gif)
